addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16515,7 +16524,7 @@ msgstr ""
 #. Description of setting "Videos -> Library -> Show All Items entry"
 #: system/settings/settings.xml
 msgctxt "#38012"
-msgid "Show \"All items\" entry in directory (for example \"All albums\" or \"All seasons\")"
+msgid "Show \"All items\" entry in directory (for example, \"All albums\" or \"All seasons\")"
```

Comment:
I think Ive managed to use **e.g.** instead without using **E.g.** maybe need to think how to do it better how overall entries are phrased.